content/news/release-20150412.md

```diff
@@ -0,0 +1,41 @@
+* A huge number of smaller gameplay fixes and improvements have been added. A few examples:
+   * After having been captured while attacking a target, defense structures will now stop firing at that target (if it's now friendly).
+   * Friendly units blocking the exit of a production building will now move out of the way if needed.
+   * Units will no longer hunt cloaked units.
+   * The spy plane in Red Alert will no longer lure your anti-air capable units across the map.
+   * Units that have been loaded into a transport will now be correctly removed from the minimap.
+   * Your faction will not be revealed to enemies when you have selected a random faction in the lobby.
+   * In sell mode, the tooltip will now show the expected refund value for structures.
+   * Harvesters should now no longer block themselves at refineries and also make use of all available refineries, if the nearest ones are occupied.
+* A similarly massive number of performance improvements have been made in this release.
+* Many race conditions and crashes have been fixed across the board.
+
+For the complete list of changes, visit [our changelog page](http://changelog.openra.net).
+
+A note on key bindings: In the previous release, the key to toggle team chat was changed to 'Alt', which, looking back, was an unfortunate decision. It has been reverted to 'Tab' in this release, where it serves a double role: on an empty line, it toggles team chat, otherwise it auto-completes player names.
+
+For modders and mappers, there have been quite a few important changes as well. We have introduced a new map format, to which maps running on this release will be updated automatically. All of the shared engine code has been moved into the OpenRA.Mods.Common project, so that third-party mods can drop their dependencies on our game mods. A lot of code has been changed to enable features that will be needed by the Tiberian Sun mod. One important change for Red Alert mappers is that the gem mine has been replaced with a new one. For more details, check out the [changelog](http://changelog.openra.net), as well as the [trait](https://github.com/openra/openra/wiki/Traits) and [Lua API](https://github.com/openra/openra/wiki/Lua-API) documentation.
+
+Looking forward, we hope to get the next release out faster than this time around (famous last words!). Feature-wise, you can probably expect the first appearance of the in-game map editor in the next release. Beyond that, we will see... If you want to keep track of the development, keep an eye on the [release milestone](https://github.com/OpenRA/OpenRA/milestones/Next%20release) and the [development changelog](https://github.com/openra/openra/wiki/Changelog-(bleed).md).
+
+We will also try to make more use of [our YouTube channel](https://www.youtube.com/channel/UCRoiPL1J4K1-EhQeNazrYig) to show off new and little known existing features, provide mission walkthroughs and so on, so watch and subscribe!
```

Comment:
Maybe add a sentence saying that we will also include the best player-submitted videos on our channel? (we really should start doing this)